website/en/docs/editors/vscode.md

```diff
@@ -10,8 +10,6 @@ integrations for using Flow.
 [Flow Language Support](https://marketplace.visualstudio.com/items?itemName=flowtype.flow-for-vscode)
 provides all the functionality you would expect — linting, intellisense, type
 tooltips and click-to-definition. It's stable and under active development.
-However, it currently lacks on-the-fly linting and only shows you type errors
-after you save a file.
```

Comment:
This is not true from [0.6.0](https://github.com/flowtype/flow-for-vscode/blob/master/CHANGELOG.md#060)